src/attrib/config.ts

```diff
@@ -10,8 +10,29 @@ export const ESSENCE_FOLDER = path.join(SOURCE_FOLDER, "/essence/attrib");
 
 // Unit files not discovered by sync.ts
 export const hardcodedDiscovery = {
-  rus: ["sbps/races/rus/unit_militia_2_rus", "upgrade/races/rus/units/upgrade_militia_3", "upgrade/races/rus/units/upgrade_militia_4"],
-  mongols: ["sbps/races/mongol/unit_khan_2_mon", "sbps/races/mongol/unit_khan_3_mon", "sbps/races/mongol/unit_khan_4_mon"],
+  rus: [
```

Comment:
I'm not a super big fan of this approach, but can image it makes the process much easier. Perhaps, once finished, we can see if we can invert it (auto loading all and stripping the once we can't support)

Reply:
I understand.  I am very unhappy with it too.  I worked on a couple different dynamic approaches first before succumbing to hardcodes.  I presented evidences in comments of why it was difficult and how much filtering out we would have to do.  The data is just night and day unclean versus the other types.

```
$ ls -Rl abilities/ | grep xml | wc -l
350
$ ls -Rl info/buff_info/ | grep xml | wc -l
102
```

450 ability files and I think 40 abilities are in my first round of scope.

Happy to work with you directly on it on Discord or see what you find.

Reply:
I'll take another shot at it.

Reply:
We did it!